website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -6,10 +6,18 @@
     % if workflow == 'pre-moderation':
         Your submission <a href="${reviewable.absolute_url}">${reviewable.title}</a>, submitted to ${reviewable.provider.name} has
         % if is_rejected:
-            not been accepted. Contributors with admin permissions may edit the ${document_type} and
-            resubmit, at which time it will return to a pending state and be reviewed by a moderator.
+            % if document_type == 'preprint':
```

Comment:
This is a very branch-y template. Maybe keep preprint and registration text completely separate to keep it cleaner?